live-examples/html-examples/inline-text-semantics/meta.json

```diff
@@ -8,6 +8,14 @@
             "title": "HTML Demo: &lt;abbr&gt;",
             "type": "tabbed"
         },
+        "kbd": {
+            "baseTmpl": "tmpl/live-tabbed-tmpl.html",
+            "exampleCode": "live-examples/html-examples/inline-text-semantics/kbd.html",
+            "cssExampleSrc": "live-examples/html-examples/inline-text-semantics/css/kbd.css",
+            "fileName": "kbd.html",
+            "title": "HTML Demo: &lt;kbd&gt;",
```

Comment:
This should be `<kbd>` if #777 gets merged before.

Reply:
Yeah, I reckon this one will merge first.